addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -217,8 +217,8 @@
             t-on-touchend="() => this.onRowTouchEnd(record)"
             t-on-touchmove="() => this.onRowTouchMove(record)"
         >
-            <td t-on-keydown.synthetic="(ev) => this.onCellKeydown(ev, group, record)" t-if="hasSelectors" class="o_list_record_selector" t-on-click.stop="() => this.toggleRecordSelection(record)" tabindex="-1">
-                <CheckBox disabled="!!props.list.editedRecord or props.list.model.useSampleModel" value="record.selected" onChange.bind="() => this.toggleRecordSelection(record)" />
+            <td t-on-keydown.synthetic="(ev) => this.onCellKeydown(ev, group, record)" t-on-keyup="(ev) => this.onCellKeyup(ev, record)" t-if="hasSelectors" class="o_list_record_selector user-select-none" t-on-click.stop="(ev) => this.toggleRecordSelection(record, ev.shiftKey)" tabindex="-1">
```

Comment:
why "user-select-none" is added here ?

Reply:
If we clicked in a checkbox and then shfit+click in a td below was causing the selection of all the text above. We don't want to select anything when clicking on a td, therefore the `user-select-none`